var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -84,9 +84,12 @@ class Elemental(CMakePackage):
     depends_on('metis +int64', when='+int64')
     depends_on('mpi')
     # Allow Elemental to build internally when using 8-byte ints
-    depends_on('scalapack', when='+scalapack ~int64_blas')
+    depends_on('scalapack', when='+scalapack blas=openblas ~int64_blas')
     extends('python', when='+python')
     depends_on('python@:2.8', when='+python')
+    depends_on('gmp')
```

Comment:
i did not see `gmp` mentioned here https://github.com/elemental/Elemental, can you please check that the built library indeed links against it? You should also see `mpc` and `mpfr` there.